update.sh

```diff
@@ -46,7 +46,8 @@ pgrep apt &>/dev/null && { echo "apt is currently running. Try again later";  ex
 # install new dependencies
 type jq &>/dev/null || {
   apt-get update
-  apt-get install -y --no-install-recommends jq
+  apt-get install -y --no-install-recommends jq tmux locales-all
```

Comment:
is `locales-all` necessary? is it very big?

Reply:
I'll investigate that. It was a quick fix for an issue with termux (missing dependency), but we can probably go with a lighter, more specific resolution

Reply:
Without the package you get the error `tmux: need UTF-8 locale (LC_CTYPE) but have ANSI_X3.4-1968` when running tmux.

However, from the [debian package description](https://packages.debian.org/sid/locales-all):

> A better alternative is to install the locales package and only select desired locales, but it can be useful on a low-memory machine because some locale files take a lot of memory to be compiled.

Reply:
I could confirm that the following works just as fine:

1. `apt update && apt install locale`
2. `echo "en_US.UTF-8 UTF-8" >> /etc/locale.gen`
3. `locale-gen`

Reply:
yes, that's better

Reply:
Implemented in e2edadf7b0c20d59b8b681ec5aa29b4462fb9e10